public/app/features/logs/utils.ts

```diff
@@ -265,3 +267,21 @@ export const checkLogsError = (logRow: LogRowModel): { hasError: boolean; errorM
 
 export const escapeUnescapedString = (string: string) =>
   string.replace(/\\r\\n|\\n|\\t|\\r/g, (match: string) => (match.slice(1) === 't' ? '\t' : '\n'));
+
+export function logRowsToReadableJson(logs: LogRowModel[]) {
+  return logs.map((log) => {
+    const fields = getDataframeFields(log).reduce<Record<string, string>>((acc, field) => {
+      acc[field.key] = field.value;
+      return acc;
+    }, {});
```

Comment:
this is complete nitpicking, feel free to ignore :) 

what do you think about doing an `Object.fromEntries`?

something like:
```ts
    const fields = Object.fromEntries(
        getDataframeFields(log).map((field) => [field.key, field.value])
    );
```

i'm personally not sure if it is nicer or worse.. WDYT?

Reply:
Hm, I understand that it might be a bit more readable, however I'd prefer to use `reduce` for not looping over the object twice (`map` + `fromEntries`). Is that fine for you?

Reply:
sure, no problem 👍